### ISSUE TITLE
CPC2 Test Data Missing Intelligiblity Score

### ISSUE THREAD

**Issue by haoshuai-orka** [claritychallenge/clarity#429]
Hello, it is said that the test set of CPC2 will also be open after the challenge. I can find the related data on https://claritychallenge.org/docs/cpc2/cpc2_download, but it seems like the ground truth intelligibility score is missing in the json files?

I would like to know where I can download these ground truth scores. Thank you!

**Comment by NylanderSym** [comment]
Ditto for CPC1. What is the recommended way of using CPC data for training and evaluation without evaluation ground truth? Of course, the training set could be split arbitrarily, but as I understand it, it might be preferable to use the pre-split train and train_indep for training and eval, resp., for reproducibility?
